tests/fixtures/config/env/post_update/test_post_update_old_parser.yml

```diff
@@ -4,6 +4,9 @@ engines:
       test_app:
         location: {path: TEST_APP_LOCATION, type: dev}
     location: {disabled: true, path: DISABLED_ENGINE_LOCATION, type: dev}
+  incomplete_engine:
```

Comment:
Because I've added a new engine in test.yml, it means the files we use for comparison after updates need to be updated as well.